Formula/c/cloud-sql-proxy.rb

```diff
@@ -0,0 +1,18 @@
+class CloudSqlProxy < Formula
+  desc "Utility for connecting securely to your Cloud SQL instances"
+  homepage "https://github.com/GoogleCloudPlatform/cloud-sql-proxy"
+  url "https://github.com/GoogleCloudPlatform/cloud-sql-proxy/archive/v2.6.1.tar.gz"
+  sha256 "b5c4fd1decb2d4091f0ae118e2df8387cbee7ea166047be12c4183fbeecf0c46"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    assert_match "cloud-sql-proxy version #{version}", shell_output("#{bin}/cloud-sql-proxy --version")
+    assert_match "could not find default credentials", shell_output("echo $(#{bin}/cloud-sql-proxy test 2>&1)")
```

Comment:
```suggestion
    assert_match "could not find default credentials", shell_output("#{bin}/cloud-sql-proxy test 2>&1")
```

Reply:
The `echo` here is used to suppress the exit 1 return status code (see other example in [kamel.rb](https://github.com/Homebrew/homebrew-core/blob/master/Formula/k/kamel.rb#L40)).
I found a better way to handle it using the second argument of `shell_output` though, thanks!

Reply:
Ah yeah, Kamel shouldn't do that either.